charts/kong/values.yaml

```diff
@@ -205,7 +205,7 @@ ingressController:
     # If not set and create is true, a name is generated using the fullname template
     name:
 
-  installCRDs: true
+  installCRDs: false
```

Comment:
Please do not change the default. A significant, maybe even the majority, of users are using Helm v2.

Reply:
Please upgrade the above description accordingly as well.

Reply:
will change back. 

sorry  - when you say "the above description" do you mean in the readme regarding the default value?

Reply:
Yes and the text in Helm 2 vs 3, where applicable.

Reply:
got it! I will remove this commit from the PR